### DISCUSSION TITLE
Bots disapearing after they die in multiplayer

### DISCUSSION THREAD

**Discussion by UVILLEATZEBUGS** [FakeFishGames/Barotrauma#13840] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Bot disappeared after I restarted the round and didn't come back.

https://youtu.be/uv4iqQ8naaU video on the bug

### Reproduction steps

1.Start a multiplayer campaign
2.Kill bot
3.Restart round 

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/13819. We're planning on hotfixing this today, sorry for the inconvenience!
